File: WeddingApp.API/Models/Place.cs
using System;
using System.Collections.Generic;

namespace WeddingApp.API.Models
{
    public class Place
    {
        public int Id { get; set; }

        public User User { get; set; }

        public int UserId { get; set; }

        public string Name { get; set; }

        public DateTime Created { get; set; }

        public string Country { get; set; }

        public string City { get; set; }

        public string Address { get; set; }

        public string Facilities { get; set; }

        public int Capacity { get; set; }

        public int Price { get; set; }

        public string InPrice { get; set; }

        public string Bonuses { get; set; }

        public string Description { get; set; }
        
        public ICollection<PhotoForPlace> Photos { get; set; }

        public ICollection<Reservation> Reservations { get; set; }

        public Place()
        {
            Photos = new HashSet<PhotoForPlace>();
            Reservations = new HashSet<Reservation>();
        }
    }
}

File: WeddingApp-SPA/src/app/_models/place.ts
import { Photo } from "./photo";

export interface Place {

    id: number;
    userId: number;
    name: string;
    country: string;
    city: string;
    address: string;
    facilities?: string;
    capacity: number;
    price: number;
    inPrice?: string;
    bonuses?: string;
    description?: string;
    photos?: Photo[];
    photoUrl?: string;
}